CONTRIBUTING.md

```diff
@@ -56,15 +56,6 @@ pull request to the master branch referencing the specific issue you addressed.
 
 ## Setup development environment
 
-1. install `poetry` (only once per machine):\
-   `curl -sSL https://install.python-poetry.org | python3 -`\
-   or [checkout installation instruction](https://python-poetry.org/docs/#installation) or
-   use [conda forge version](https://anaconda.org/conda-forge/poetry)
-1. (Optional, skip if not sure) Disable automatic environment creation:\
-   `poetry config virtualenvs.create false`
-1. (Optional) Install the optional dependency
-1. install all dependencies in one command (have to be run in thibe project directory):\
-   `poetry install`
 1. install `pre-commit` hooks to git repo:\
    `pre-commit install`
 1. you are ready to code!
```

Comment:
"pre-commit install" will also run "pip install -e ."? Or should we add this step?